docker-compose.yml

```diff
@@ -11,20 +11,25 @@ app:
     - DATABASE_URL=postgres://tock_user@db/tock
     - RUNNING_IN_DOCKER=yup
     - DJANGO_SETTINGS_MODULE=tock.settings.dev
-  command: "python manage.py runserver 0.0.0.0:${EXTERNAL_PORT}"
+  command: "python manage.py runserver 0.0.0.0:1234"
 db:
   image: postgres:9.4
   environment:
     - POSTGRES_DB=tock
     - POSTGRES_USER=tock_user
 web:
-  image: nginx
+  image: nginx:1.11
```

Comment:
Yep!  Specificaly, the `envsubst` command used to fill-in the values for the environment variables in the config template is only present on more recent builds/versions of the nginx container.